tests/Microsoft.CodeAnalysis.Testing/Microsoft.CodeAnalysis.CodeFix.Testing.UnitTests/AdditionalFilesFixTests.cs

```diff
@@ -77,7 +77,7 @@ public async Task TestMarkupDiagnosticFixedByAddingAdditionalFileFailsIfTextInco
                 }.RunAsync();
             });
 
-            Assert.Equal("content of 'File1.txt' was expected to be 'Wrong file text' but was 'File text'", exception.Message);
+            Assert.Equal($"content of 'File1.txt' did not match. Diff shown with expected as baseline:{Environment.NewLine}- Wrong file text{Environment.NewLine}+ File text{Environment.NewLine}", exception.Message);
```

Comment:
💡 It seems like a verbatim string here would improve readability, but this could work too.

Reply:
That solves the appveyor LF problem, but when you build and test on linux, it will still fail because `core.autocrlf=true` means LF on linux.